nixos/modules/services/hardware/bolt.nix

```diff
@@ -20,15 +19,13 @@ with lib;
         '';
       };
 
+      package = mkPackageOption pkgs "bolt" { };
```

Comment:
Noting that https://nixos.org/manual/nixos/unstable/#sec-option-declarations-util says:

> You shouldn’t necessarily make package options for all of your modules. You can always overwrite a specific package throughout nixpkgs by using [nixpkgs overlays](https://nixos.org/manual/nixpkgs/stable/#chap-overlays).

This doesn't seem to imply "don't", and the declarative `package` option is still nicer